scripts/renumber_ants.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+"""
+A command-line script for renumbering antenna numbers > 255 if possible in uvfits files.
+"""
+import numpy as np
+import os
+import argparse
+from pyuvdata import UVData
+
+# setup argparse
+a = argparse.ArgumentParser(description="A command-line script for renumbering antenna numbers > 255 if possible in uvfits files.")
+a.add_argument("file_in", type=str, help="input uvfits file.")
+a.add_argument("file_out", type=str, help="output uvfits file.")
+a.add_argument("--overwrite", default=False, action='store_true', help="overwrite output file if it already exists.")
+a.add_argument("--verbose", default=False, action='store_true', help="report feedback to stdout.")
+
+# get args
+args = a.parse_args()
+
+if os.path.exists(args.file_out) and args.overwrite is False:
+    print("{} exists, not overwriting...".format(args.file_out))
+    continue
+
+uv_obj = UVData()
+uv_obj.read_uvfits(args.file_in)
+
+large_ant_nums = uv_obj.antenna_numbers[np.where(uv_obj.antenna_numbers > 255)[0]]
+
+new_nums = sorted(list(set(range(256)) - set(uv_obj.antenna_numbers)))
```

Comment:
It needs to be range(255) I think because 0-based antenna 255 is 1-based 256 and that gets turned into 0 in some 8-bit code path in CASA

Reply:
Ok, I'll decrease the maximum antenna number to 254.